rpcserver.go

```diff
@@ -6815,6 +6815,12 @@ func (r *rpcServer) UpdateChannelPolicy(ctx context.Context,
 			minTimeLockDelta)
 	}
 
+	if req.TimeLockDelta > uint32(MaxTimeLockDelta) {
+		return nil, fmt.Errorf("time lock delta of %v is too big, "+
```

Comment:
nit: use `%d` for numeric types in format strings.

Reply:
> nit: use `%d` for numeric types in format strings.

I found many occurence of code like this:https://github.com/lightningnetwork/lnd/blob/628fc017324daf57b822a2c2b7c5c763f3131736/rpcserver.go#L6788-L6822

All of them are using `%v` for numeric types in error format, I think `%v` is better than `%d` in error message, because `%v` contains variable type.